cli/src/android/add.ts

```diff
@@ -20,7 +22,7 @@ export async function addAndroid(config: Config) {
     },
   );
 
-  await runTask(`Syncing Gradle`, async () => {
+  await runTask(`Syncing Gradle with ${c.input('./gradlew')}`, async () => {
```

Comment:
Thanks, updated. 

I think we should show it because we do something similar for iOS (`Updating iOS native dependencies with pod install`), and it aligns with the goal that the Capacitor CLI should be more transparent/informative.